test/cluster.sh

```diff
@@ -60,6 +60,19 @@ function create_monitoring() {
     -f config/monitoring/tracing/zipkin
 }
 
+function create_namespace() {
```

Comment:
can we pass in the actual name as parameter? so that other tests can use it but with their choice of namespace name.